src/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './DevopnessApiClient';
-export * from './api/generated/models';
+export * as DevopnessSdkExceptions from './common/Exceptions';
```

Comment:
Same here, I think that we could drop the `Devopness*` prefix. The usage would be already quite clear if it looked something like:
```
import { SdkModels, SdkExceptions } from `devopness-sdk-js`
const project: SdkModels.Project;
```
or
```
import * as Devopness from `devopness-sdk-js`
const project: Devopness.SdkModels.Project;
```